docs/ibc/light-clients/client-state.md

```diff
@@ -28,6 +28,8 @@ implementer is in charge of determining which checks are required. See the [Tend
 - A `Frozen` status indicates that misbehaviour was detected in the counterparty chain and the client is not allowed to be used.
 - An `Expired` status indicates that a client is not allowed to be used because it was not updated for longer than the trusting period.
 - An `Unknown` status indicates that there was an error in determining the status of a client.
+- An `Unauthorized` status indicates that usage of the client is not allowed.
```

Comment:
The formatting of the rendered text looks a bit weird with the `- >`. Was its use intentional?

Reply:
this was intentional to add a `note` section, but I think Colin brought up a good point that this is not the best place for these docs.